### PR TITLE
chore(kubernetes_logs source): Correct docs type and alter example for extra_field_selector

### LINKED ISSUE #6140: Update helm docs `extra_field_selector` to reference correct type and usage
As seen in #6130 our documentation around the extra_field_selector is unclear. 

Currently the documenation shows this field as supporting an array which is incorrect. This field is a kubernetes-formatted filter predicate string. 

When we update the examples we should include a multiform example like

`metadata.namespace!=other,metadata.name=mypod,metadata`

### PR DESCRIPTION
**Pull request by eeyun**
Closes #6140 

Signed-off-by: Ian Henry <ianjhenry00@gmail.com>

### CODE REVIEW

File: docs/reference/components/sources/kubernetes_logs.cue [hunk 1]
```diff
@@ -170,8 +170,8 @@ components: sources: kubernetes_logs: {
 				"""
 			required: false
 			type: array: {
```
Comment (binarylogic) [inline docs/reference/components/sources/kubernetes_logs.cue:172]:
```suggestion
			type: string: {
```

File: docs/reference/components/sources/kubernetes_logs.cue [hunk 2]
```diff
@@ -170,8 +170,8 @@ components: sources: kubernetes_logs: {
...
-				default: []
-				items: type: string: examples: ["metadata.name!=pod-name-to-exclude"]
+				default: ""
+				items: type: string: examples: "metadata.name!=pod-name-to-exclude,metadata.name=mypod"
```
Comment (binarylogic) [inline docs/reference/components/sources/kubernetes_logs.cue:174]:
```suggestion
				default: ""
				examples: ["metadata.name!=pod-name-to-exclude,metadata.name=mypod"]
```

Comment (eeyun) [reply docs/reference/components/sources/kubernetes_logs.cue:174]: So in this case, this entry is specifically a string and not an array - I'm assuming these square brackets are a cue notation thing?

Comment (MOZGIII) [reply docs/reference/components/sources/kubernetes_logs.cue:174]: Yes, and we can pass more than one example - which I suggest we do

File: docs/reference/components/sources/kubernetes_logs.cue [hunk 3]
```diff
@@ -182,8 +182,8 @@ components: sources: kubernetes_logs: {
 				"""
 			required: false
 			type: array: {
```
Comment (MOZGIII) [inline docs/reference/components/sources/kubernetes_logs.cue:184]:
```suggestion
			type: string: {
```

File: docs/reference/components/sources/kubernetes_logs.cue [hunk 4]
```diff
@@ -182,8 +182,8 @@ components: sources: kubernetes_logs: {
...
-				default: []
-				items: type: string: examples: ["my_custom_label!=my_value"]
+				default: ""
+				items: type: string: examples: "my_custom_label!=my_value,my_other_custom_label=my_value"
```
Comment (MOZGIII) [inline docs/reference/components/sources/kubernetes_logs.cue:186]:
```suggestion
				items: type: string: examples: ["my_custom_label!=my_value", "my_custom_label!=my_value,my_other_custom_label=my_value"]
```

**Review by MOZGIII — APPROVED** [review]

**Pull request merged**